.ropeproject/config.py

```diff
@@ -0,0 +1,112 @@
+# The default ``config.py``
```

Comment:
Was the addition of this file intentional? (Don't mind, don't care, just making sure :wink:)